docs/source/cheatsheet.md

```diff
@@ -12,9 +12,10 @@ html_meta:
 - https://facelessuser.github.io/pymdown-extensions/
```

Comment:
@stevepiercy How about removing this file completely in favour of? https://github.com/plone/documentation/blob/6-dev/docs/contributing/writing-docs-guide.md

Reply:
I agree with @ksuess. Keeping the file is useful to authors.

We could synchronize content between the two files, and exclude the Volto cheatsheet from the Plone 6 Docs build.